manager/controllers/advisory_detail.go

```diff
@@ -97,12 +97,13 @@ func AdvisoryDetailHandler(c *gin.Context) {
 	c.JSON(http.StatusOK, &resp)
 }
 
-func parseCVEs(jsonb *postgres.Jsonb) ([]string, error) {
+func parseCVEs(jsonb []byte) ([]string, error) {
```

Comment:
oh, it looks a bit dangerous. Isn't it possible to use postgres dialect with new gorm?

Reply:
New postgres dialect dropped support form JSONB, we can use custom made type for JSONB.